platforms/documentation/docs/src/docs/userguide/authoring-builds/tasks/build_services.adoc

```diff
@@ -73,41 +73,30 @@ Note that you should *not* implement the link:{javadocPath}/org/gradle/api/servi
 A build service implementation must be thread-safe, as it will potentially be used by multiple tasks concurrently.
 
 [[using_a_build_service_from_a_task]]
```

Comment:
@lkasso I felt the existing documentation had some unnecessary duplication. I merged the sections about registering and consuming build services, and promoted `@ServiceReference` as preferred over `@Internal` properties.

Reply:
I like this update a lot.